examples/plot_wlasso.py

```diff
@@ -10,90 +10,93 @@
 
 # Authors: Quentin Bertrand <quentin.bertrand@inria.fr>
 #          Quentin Klopfenstein <quentin.klopfenstein@u-bourgogne.fr>
-#
+#          Mathurin Massias
+#          Kenan Sehic
 # License: BSD (3-clause)
 
-import time
-
 import numpy as np
 
-from sklearn.datasets import make_regression
-from sklearn.model_selection import train_test_split
+from celer import Lasso, LassoCV
 
-from celer import Lasso
+from sklearn.datasets import make_regression
 
 from sparse_ho.models import wLasso
 from sparse_ho.criterion import CV
 from sparse_ho.implicit_forward import ImplicitForward
 from sparse_ho.utils import Monitor
 from sparse_ho.ho import grad_search
-from sparse_ho.datasets import get_data
 
-print(__doc__)
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import mean_squared_error
 
-dataset = 'rcv1_train'
-# dataset = 'simu'
+# from wlasso_hpo import sparse_ho_model, random_search
 
-if dataset != 'simu':
-    X_train, X_val, X_test, y_train, y_val, y_test = get_data(dataset)
-    X_train = X_train[:, :1000]
-    X_test = X_test[:, :1000]
-else:
-    X, y = make_regression(n_samples=100, n_features=100, noise=1)
-    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.33)
-    X_train, X_val, y_train, y_val = train_test_split(
-        X_train, y_train, test_size=0.5)
 
-n_samples, n_features = X_train.shape
+X, y = make_regression(n_samples=500, n_features=600, noise=40, random_state=2)
```

Comment:
I haven't checked how the noise level was set. It may be better to use noise = 0 and to add nosie ourservles by setting a SNR of 5, wdyt?

Reply:
perfect, I put the noise 0